Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -737,7 +737,7 @@ bool RenderLayer::willCompositeClipPath() const
     if (renderer().hasMask())
         return false;
 
-    return (clipPath->type() != PathOperation::Shape || clipPath->type() == PathOperation::Shape) && GraphicsLayer::supportsLayerType(GraphicsLayer::Type::Shape);
+    return clipPath->type() != PathOperation::Shape || (clipPath->type() == PathOperation::Shape && GraphicsLayer::supportsLayerType(GraphicsLayer::Type::Shape));
```

Comment:
Can't this be
```
return clipPath->type() != PathOperation::Shape || GraphicsLayer::supportsLayerType(GraphicsLayer::Type::Shape));
```

Reply:
Done, my bigger point was though that I did not know which of the two ways stated above was the intention :)